mocha-client-tests/addon/tests/lib/background-messaging.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function(done) {
+            chrome.runtime.sendMessage({action: 'ping'},function(response) {
+                expect(response).to.equal('pong');
```

Comment:
when this assertion fails, it only gets logged to the browser console. Can you make it show up in the mocha test output instead? For example, if you change `expect(response).to.equal('pong')` to `expect(response).to.equal('nope')` then I see this confusing timeout instead of the assertion error:

![screenshot 2016-11-07 11 25 33](https://cloud.githubusercontent.com/assets/55398/20068082/1514c772-a4dd-11e6-99b3-5130cb4afb9a.png)